### PR TITLE
Update ReadMe file with missing refactor plugins in apidom-ns-openapi-3-1

### PR DESCRIPTION
**Pull request by cka121**
This PR updates the README file of apidom-ns-openapi-3-1 package with documentation of 
`normalize-discriminator-mapping (POC)`, `normalize-parameter-examples` and `normalize-header-examples`.
[#OSS-10](https://smartbear.atlassian.net/browse/OSS-10)

### My PR contains...

- [X] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [ ] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/apidom-ns-openapi-3-1/README.md [hunk 1]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+//       }
+//     }
+//   },
+//   '$$my-normalized': { 'parameter-examples': [ '/paths/~1/get/parameters/0' ] }
+// }
+```
+
+#### Normalize Header examples plugin
+
+`normalize-header-examples` plugin enforces the precedence rule: header-level example / examples override the example(s) declared by the header's schema.The plugin only overrides existing schema.example / schema.examples (it does not create them) and records normalized headers to avoid repeated changes.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: Similar as in the parameters plugin, not sure if this is better:

```suggestion
`Header.examples` and `header.example` override `header.schema.examples` and `header.schema.example` fields. The plugin overrides only the existing `header.schema.examples` and `header.schema.example`.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 2]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+//   }
+// }
+```
+
+#### Normalize Discriminator mapping plugin
+This plugin normalizes the `discriminator.mapping` field in a Schema Object by:
+
+- Converting mapping values into inline Schema Objects when possible.
+- Adding missing mapping entries based on the schema's `oneOf`, `anyOf`, or a prepared `allOf` mapping.
+- Storing the normalized mapping under `discriminator.x-normalized-mapping`..
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- Storing the normalized mapping under `discriminator.x-normalized-mapping`.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 3]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
 //   }
 // }
 ```
+#### Normalize Parameter examples plugin
+
+`normalize-parameter-examples` plugin enforces the precedence rule: parameter-level examples take priority over schema-level examples. 
+
+- If a parameter defines its own `example` or `examples`, these will override any corresponding values present in the parameter's schema.
+- The plugin will only override `example` or `examples` fields if they already exist on the schema; it will not add new example fields to schemas that do not already declare them.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- The plugin will only override `example` or `examples` fields if they already exist in the schema; it will not add new example fields to schemas that do not already declare them.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 4]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Targets: `Parameter` objects that reference a `Schema` element.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: I think this can be removed, as it is already mentioned in the first sentence.

File: packages/apidom-ns-openapi-3-1/README.md [hunk 5]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Skips: parameters declared under `components` (no changes to reusable component definitions).
+- Preference: `parameter.examples` (map of named Example objects) wins over `parameter.example` (deprecated single value).
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- `parameter.examples` has precedence over deprecated `parameter.example`.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 6]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
 //   }
 // }
 ```
+#### Normalize Parameter examples plugin
+
+`normalize-parameter-examples` plugin enforces the precedence rule: parameter-level examples take priority over schema-level examples.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: Not sure if this sounds better, maybe others can weigh in here:
```suggestion
`Parameter.examples` and `parameter.example` override `parameter.schema.examples` and `parameter.schema.example` fields. The plugin overrides only the existing `parameter.schema.examples` and `parameter.schema.example`.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 7]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
 // }
 ```
+#### Normalize Parameter examples plugin
+
+`normalize-parameter-examples` plugin enforces the precedence rule: parameter-level examples take priority over schema-level examples. 
+
+- If a parameter defines its own `example` or `examples`, these will override any corresponding values present in the parameter's schema.
+- The plugin will only override `example` or `examples` fields if they already exist on the schema; it will not add new example fields to schemas that do not already declare them.
+- Targets: `Parameter` objects that reference a `Schema` element.
+- Skips: parameters declared under `components` (no changes to reusable component definitions).
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- Does not apply to parameters defined under `components`.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 8]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Preference: `parameter.examples` (map of named Example objects) wins over `parameter.example` (deprecated single value).
+- Idempotent: records normalized parameters to avoid re-applying changes.
+- Storage: normalization state is stored on the root `OpenApi3_1Element` under a field name given by `storageField` (default `x-normalized`).
+- The plugin only overrides schema examples when the schema already declares them. If `schema` lacks `example` and `examples`, the plugin does nothing for that parameter.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: This is already mentioned in the second point.

File: packages/apidom-ns-openapi-3-1/README.md [hunk 9]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+//   },
+//   '$$my-normalized': { 'parameter-examples': [ '/paths/~1/get/parameters/0' ] }
+// }
+```
+
+#### Normalize Header examples plugin
+
+`normalize-header-examples` plugin enforces the precedence rule: header-level example / examples override the example(s) declared by the header's schema.The plugin only overrides existing schema.example / schema.examples (it does not create them) and records normalized headers to avoid repeated changes.
+
+- Targets: Header objects that reference a Schema element.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: Already mentioned in the first sentence.

File: packages/apidom-ns-openapi-3-1/README.md [hunk 10]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Skips: headers defined under components (to avoid mutating shared definitions).
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- Does not apply to headers defined under `components`.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 11]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Preference: header.examples (map of Example objects) wins over header.example (deprecated).
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- `header.examples` has precedence over deprecated `header.example`.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 12]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+//     'header-examples': [ '/paths/~1/get/responses/200/headers/content-type' ]
+//   }
+// }
+```
+
+#### Normalize Discriminator mapping plugin
+This plugin normalizes the `discriminator.mapping` field in a Schema Object by:
+
+- Converting mapping values into inline Schema Objects when possible.
+- Adding missing mapping entries based on the schema's `oneOf`, `anyOf`, or a prepared `allOf` mapping.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- Adding missing mapping entries based on the schema's `oneOf`, `anyOf`, or a prepared `allOf` mapping. The `allOf` mapping is created based on the schemas defined in `components.schemas` during dereferencing.
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 13]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Storing the normalized mapping under `discriminator.x-normalized-mapping`..
+- Idempotent: processed schemas are recorded and skipped on subsequent runs.
+
+This plugin is intended to run on dereferenced OpenAPI 3.1 documents (the dereferencer is expected to annotate candidate schema items with meta properties such as schemaName, ref-fields and ref-origin).
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: Might also need input from others:

```suggestion
The `discriminator.mapping` field is not modified by the plugin.

This plugin is intended to run on dereferenced OpenAPI 3.1 documents. During dereferencing Schema Objects are annotated with meta properties and the `allOf` mapping is created for Schema Objects defined in `components.schemas`. 
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 14]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+      }
+    }
+  }
+}
+```
+
+```js 
+const uri = 'path/to/fixture-example.json'; // the arbitrary file name shown above
+
+// 1) dereference the document (dereferencer should annotate ref metadata)
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
// 1) dereference the document to annotate schemas with required metadata
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 15]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+const dereferenced = await dereference(uri, {
+  parse: { mediaType: mediaTypes.latest('json') },
+  resolve: {
+    baseURI: uri,
+    resolvers: [ new FileResolver({ fileAllowList: [/\.json$/] }) ],
+  },
+  dereference: { strategyOpts: { 'openapi-3-1': { dereferenceDiscriminatorMapping: true } } },
+});
+
+// 2) run the plugin and pass the same filename as baseURI
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
// 2) dispatch the plugin and pass the same baseURI
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 16]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+#### Normalize Parameter examples plugin
+
+`normalize-parameter-examples` plugin enforces the precedence rule: parameter-level examples take priority over schema-level examples. 
+
+- If a parameter defines its own `example` or `examples`, these will override any corresponding values present in the parameter's schema.
+- The plugin will only override `example` or `examples` fields if they already exist on the schema; it will not add new example fields to schemas that do not already declare them.
+- Targets: `Parameter` objects that reference a `Schema` element.
+- Skips: parameters declared under `components` (no changes to reusable component definitions).
+- Preference: `parameter.examples` (map of named Example objects) wins over `parameter.example` (deprecated single value).
+- Idempotent: records normalized parameters to avoid re-applying changes.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: NOTE/IDEA here: other plugins do not mention that they are idempotent, as the documentation was not updated when that enhancement was added. Perhaps we should mention it in the overall plugins section?

File: packages/apidom-ns-openapi-3-1/README.md [hunk 17]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Storage: normalization state is stored on the root `OpenApi3_1Element` under a field name given by `storageField` (default `x-normalized`).
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: Another NOTE/IDEA here: this normalization field was added to make the plugins idempotent. As with the suggestion for idempotence, perhaps we should mention it, together with idempotence, for the overall plugins description? And here we could just leave that it is stored under `<storageField>.parameter-examples`? 

Otherwise, suggestion:

```suggestion
- Normalization state is stored in the root `OpenApi3_1Element` in `<storageField>.parameter-examples` field. `<storageField>` can be customized in the plugin configuration (default: `x-normalized`).
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 18]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+
+#### Normalize Header examples plugin
+
+`normalize-header-examples` plugin enforces the precedence rule: header-level example / examples override the example(s) declared by the header's schema.The plugin only overrides existing schema.example / schema.examples (it does not create them) and records normalized headers to avoid repeated changes.
+
+- Targets: Header objects that reference a Schema element.
+- Skips: headers defined under components (to avoid mutating shared definitions).
+- Preference: header.examples (map of Example objects) wins over header.example (deprecated).
+- Idempotent: records normalized headers to avoid re-applying changes.
+- Storage: normalization state is stored on the root `OpenApi3_1Element` under a field name given by `storageField` (default `x-normalized`).
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
- Normalization state is stored in the root `OpenApi3_1Element` in `<storageField>.header-examples` field. `<storageField>` can be customized in the plugin configuration (default: `x-normalized`).
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 19]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+// }
+```
+
+#### Normalize Discriminator mapping plugin
+This plugin normalizes the `discriminator.mapping` field in a Schema Object by:
+
+- Converting mapping values into inline Schema Objects when possible.
+- Adding missing mapping entries based on the schema's `oneOf`, `anyOf`, or a prepared `allOf` mapping.
+- Storing the normalized mapping under `discriminator.x-normalized-mapping`..
+- Idempotent: processed schemas are recorded and skipped on subsequent runs.
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: Do we want to mention the storage field as in the previous plugins, so `<storageField>.discriminator-mapping`?

File: packages/apidom-ns-openapi-3-1/README.md [hunk 20]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- The plugin will only override `example` or `examples` fields if they already exist on the schema; it will not add new example fields to schemas that do not already declare them.
+- Targets: `Parameter` objects that reference a `Schema` element.
+- Skips: parameters declared under `components` (no changes to reusable component definitions).
+- Preference: `parameter.examples` (map of named Example objects) wins over `parameter.example` (deprecated single value).
+- Idempotent: records normalized parameters to avoid re-applying changes.
+- Storage: normalization state is stored on the root `OpenApi3_1Element` under a field name given by `storageField` (default `x-normalized`).
+- The plugin only overrides schema examples when the schema already declares them. If `schema` lacks `example` and `examples`, the plugin does nothing for that parameter.
+
+```js 
+import { OpenApi3_1Element, refractorPluginNormalizeParameterExamples } from '@swagger-api/apidom-ns-openapi-3-1';
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md:444]:
```suggestion
import { toValue } from '@swagger-api/apidom-core';
import { OpenApi3_1Element, refractorPluginNormalizeParameterExamples } from '@swagger-api/apidom-ns-openapi-3-1';
import { parse } from '@swagger-api/apidom-parser-adapter-yaml-1-2';
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 21]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+
+// default
+const yamlDefinition: `
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const yamlDefinition = `
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 22]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+- Skips: parameters declared under `components` (no changes to reusable component definitions).
+- Preference: `parameter.examples` (map of named Example objects) wins over `parameter.example` (deprecated single value).
+- Idempotent: records normalized parameters to avoid re-applying changes.
+- Storage: normalization state is stored on the root `OpenApi3_1Element` under a field name given by `storageField` (default `x-normalized`).
+- The plugin only overrides schema examples when the schema already declares them. If `schema` lacks `example` and `examples`, the plugin does nothing for that parameter.
+
+```js 
+import { OpenApi3_1Element, refractorPluginNormalizeParameterExamples } from '@swagger-api/apidom-ns-openapi-3-1';
+
+// default
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion

```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 23]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+            name: idempotent
+            schema:
+              type: number
+              examples: [1]
+            examples:
+              example1:
+                value: 2`
+
+const apiDOM = await parse(yamlDefinition);
+const openApiElement = OpenApi3_1Element.refract(parseResult, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElement = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 24]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+//             examples: { example1: { value: 2 } }
+//           }
+//         ]
+//       }
+//     }
+//   },
+//   'x-normalized': { 'parameter-examples': [ '/paths/~1/get/parameters/0' ] }
+// }
+
+// custom storage field name
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md:489]: Not sure if we would want this to have in a separate code block, e.g.
```suggestion
// code block end with ```

Defining a custom storage field name:

// code block start
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 25]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+const yamlDefinition: `
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const yamlDefinition = `
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 26]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+          schema:
+            type: number
+            example: 1
+          examples:
+            example1:
+              value: 2
+`
+
+const apiDOM = await parse(yamlDefinition);
+const openApiElement = OpenApi3_1Element.refract(parseResult, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElement = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 27]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+  plugins: [refractorPluginNormalizeParameterExamples({ storageField: '$$my-normalized' })],
+});
+toValue(openApiElement);
+const element2 =
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion

```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 28]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+`normalize-header-examples` plugin enforces the precedence rule: header-level example / examples override the example(s) declared by the header's schema.The plugin only overrides existing schema.example / schema.examples (it does not create them) and records normalized headers to avoid repeated changes.
+
+- Targets: Header objects that reference a Schema element.
+- Skips: headers defined under components (to avoid mutating shared definitions).
+- Preference: header.examples (map of Example objects) wins over header.example (deprecated).
+- Idempotent: records normalized headers to avoid re-applying changes.
+- Storage: normalization state is stored on the root `OpenApi3_1Element` under a field name given by `storageField` (default `x-normalized`).
+
+```js 
+import { OpenApi3_1Element, refractorPluginNormalizeHeaderExamples } from '@swagger-api/apidom-ns-openapi-3-1';
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md:524]:
```suggestion
import { toValue } from '@swagger-api/apidom-core';
import { OpenApi3_1Element, refractorPluginNormalizeHeaderExamples } from '@swagger-api/apidom-ns-openapi-3-1';
import { parse } from '@swagger-api/apidom-parser-adapter-yaml-1-2';
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 29]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+
+// default
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion

```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 30]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+            content-type:
+              schema:
+                type: number
+                example: 1
+              examples:
+                example1:
+                  value: 2
+`
+const apiDOM = await parse(yamlDefinition);
+const openApiElement = OpenApi3_1Element.refract(parseResult, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElement = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 31]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+              schema:
+                type: number
+                example: 1
+              examples:
+                example1:
+                  value: 2
+`
+
+const apiDOM = await parse(yamlDefinition);
+const openApiElement = OpenApi3_1Element.refract(parseResult, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElement = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 32]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+   plugins: [refractorPluginNormalizeHeaderExamples({ storageField: '$$normalized' })],
+});
+toValue(openApiElement);
+const element2 =
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion

```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 33]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+        "200":
+          headers:
+            content-type:
+              schema:
+                type: number
+                example: 1
+              examples:
+                example1:
+                  value: 2
+`
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]: Needs a proper closing tag ```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 34]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+//       }
+//     }
+//   },
+//   'x-normalized': {
+//     'header-examples': [ '/paths/~1/get/responses/200/headers/content-type' ]
+//   }
+// }
+
+// custom storage field name
+const yamlDefinition: `
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const yamlDefinition = `
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 35]
```diff
@@ -431,6 +431,405 @@ toValue(openApiElement);
...
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+```js 
+const uri = 'path/to/fixture-example.json'; // the arbitrary file name shown above
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md:691]: Missing imports:

```suggestion
import { toValue, dispatchRefractorPlugins } from '@swagger-api/apidom-core';
import { dereference } from '@swagger-api/apidom-reference';
import FileResolver from '@swagger-api/apidom-reference/resolve/resolvers/file';
import {
  createToolbox,
  refractorPluginNormalizeDiscriminatorMapping,
  keyMap,
  getNodeType,
  mediaTypes,
} from '@swagger-api/apidom-ns-openapi-3-1';

const uri = 'path/to/fixture-example.json'; // the arbitrary file name shown above
```

File: packages/apidom-ns-openapi-3-1/README.md (hunk 6 above)
Comment (robert-hebel-sb) [reply packages/apidom-ns-openapi-3-1/README.md]: I'm for this change looks more aligned to the previous plugins description

File: packages/apidom-ns-openapi-3-1/README.md (hunk 33 above)
Comment (cka121) [reply packages/apidom-ns-openapi-3-1/README.md]: Actually the closing tag for this example is after custom storage field example.

File: packages/apidom-ns-openapi-3-1/README.md [hunk 36]
```diff
@@ -431,6 +432,375 @@ toValue(openApiElement);
...
+              type: number
+              examples: [1]
+            examples:
+              example1:
+                value: 2`
+
+const apiDOM = await parse(yamlDefinition);
+
+//default 
+const openApiElement = OpenApi3_1Element.refract(apiDom.result, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElement = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 37]
```diff
@@ -431,6 +432,375 @@ toValue(openApiElement);
...
+//           }
+//         ]
+//       }
+//     }
+//   },
+//   'x-normalized': { 'parameter-examples': [ '/paths/~1/get/parameters/0' ] }
+// }
+
+// custom storage field name
+const openApiElementWithCustomField = OpenApi3_1Element.refract(apiDom.result, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElementWithCustomField = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 38]
```diff
@@ -431,6 +432,375 @@ toValue(openApiElement);
...
+                type: number
+                example: 1
+              examples:
+                example1:
+                  value: 2
+`
+const apiDOM = await parse(yamlDefinition);
+
+// default
+const openApiElement = OpenApi3_1Element.refract(apiDom.result, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElement = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 39]
```diff
@@ -431,6 +432,375 @@ toValue(openApiElement);
...
+//     }
+//   },
+//   'x-normalized': {
+//     'header-examples': [ '/paths/~1/get/responses/200/headers/content-type' ]
+//   }
+// }
+
+// custom storage field name
+
+const openApiElementWithCustomField = OpenApi3_1Element.refract(apiDom.result, {
```
Comment (glowcloud) [inline packages/apidom-ns-openapi-3-1/README.md]:
```suggestion
const openApiElementWithCustomField = OpenApi3_1Element.refract(apiDOM.result, {
```

File: packages/apidom-ns-openapi-3-1/README.md [hunk 40]
```diff
@@ -431,6 +432,375 @@ toValue(openApiElement);
...
+// {
+//   openapi: '3.1.0',
+//   paths: {
+//     '/': {
+//       get: {
+//         parameters: [
+//           {
+//             in: 'query',
+//             name: 'idempotent',
+//             schema: { type: 'number', examples: [ 2 ] },
```
Comment (robert-hebel-sb) [inline packages/apidom-ns-openapi-3-1/README.md]: nit:
```suggestion
//             schema: { type: 'number', examples: [2] },
```
